### PR TITLE
Change id of reference to match IGV locus search. Fix #4013.

### LINKED ISSUE #4013: Locus search box on RNA view IGV.js for hg38 not working
Perhaps needs an update to properly pass on the build, as with the DNA view.

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

Fix #4013 - build now called hg38 instead of GRCh38 at IGV. 🙄

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [x] tests executed by DN

### CODE REVIEW

**Comment by dnil** [comment]
✅ Tested on stage.

Before:
Screenshot 2023-07-03 at 16 01 10

After: 
Screenshot 2023-07-03 at 16 40 37

**Review by northwestwitch — APPROVED** [review]
Nice! 👍🏻

File: requirements.txt [hunk 1]
```diff
@@ -35,7 +35,7 @@ flask-ldapconn
 cyvcf2
 configobj
 ped_parser
-pydantic
+pydantic<2
```
Comment (northwestwitch) [inline requirements.txt:38]: 💯

**Comment by dnil** [comment]
Thanks for the review! You are supposed to be on vacation though.. 😁 

I dunno, kind of feel this is not a biggie that would need a summer hotfix. Lets leave it here until we are back - or someone complains compellingly - and merge then?

**Comment by dnil** [comment]
Or, yeah, lets merge so we get the pydantic fix in for easy hot fixes later, but lets wait with releasing.

**Comment by northwestwitch** [comment]
> I dunno, kind of feel this is not a biggie that would need a summer hotfix. Lets leave it here until we are back - or someone complains compellingly - and merge then?

Hehe weren't you also on vacation? Yes. let's leave it like this and hope that nobody has problems during summer!

**Comment by northwestwitch** [comment]
Or let's merge 😆

**Comment by dnil** [comment]
Formally from tomorrow, so Im good. Also it's raining. 😁

**Pull request merged**